manifests/example-config.yaml

```diff
@@ -114,3 +130,33 @@ data:
       # re-advertisement outside of the immediate autonomous system,
       # but people don't usually recognize its numerical value. :)
       no-export: 65535:65281
+    bfd-profiles:
+    # The unique name to associate the BFD profile to, and to reference it
+    # from a peer entry. BFD is only available in FRR mode.
+    - name: bfdprofilename
+      # (optional) the minimum interval that this system is capable of 
+      # receiving control packets in milliseconds. The default value is 300 milliseconds
+      receive-interval: 280
```

Comment:
Pardon me for not double-checking first, but don't do these BFD time intervals miss time units (e.g. "280ms") like in keepalive-time that is "30s"?

Reply:
nope, it's an int meant to be in ms. We can probably fix this discrepancy when we move to CRDs.